bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/discovery/SmartHomeDevicesDiscovery.java

```diff
@@ -135,7 +128,6 @@ void stopScanJob() {
             startScanStateJob = null;
         }
         super.stopScan();
-        removeOlderResults(getTimestampOfLastScan());
```

Comment:
This also makes troubles, it removes Things from other instances, so I removed it